src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -6,6 +6,7 @@
  * а hashCode() был бы согласованным с реализацией equals().</p>
  */
 public class Rectangle {
+    private static final int HASHCODENUMBER = 52;
```

Comment:
let's use prime numbers

Reply:
You are right, a number divisible by 2 is not the best choice. The best choice is 31)